privacy-protections/amp/index.html

```diff
@@ -0,0 +1,64 @@
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>AMP Links</title>
+
+  <style>
+    .expected {
+      color: grey;
+      font-size: 14px;
+      margin-top: -10px;
+      margin-bottom: -4px;
+    }
+  </style>
+</head>
+<body>
+  <p><a href="../../">[Home]</a> ↣ <a href="../">[Privacy Protections Tests]</a> ↣ <strong>[AMP Links]</strong></p>
+  
+  <p>This test page has sample links to use for AMP link protection. When clicking on the sample links you should be taken to the listed "Expected" page.</P>
+
+  <div id="demo">
+    <p>google.com AMP Links</p>
+    <ul>
+      <li>
+        <p><a id="link1" href="https://www.google.com/amp/s/www.vox.com/platform/amp/identities/22530103/asians-americans-wealth-income-gap-crazy-rich-model-minority">Simple link</a></p>
+        <p class="expected">Expected: https://www.vox.com/platform/amp/identities/22530103/asians-americans-wealth-income-gap-crazy-rich-model-minority</p>
+      </li>
+      <li>
+        <p><a href="https://www.google.com/amp/s/www.nytimes.com/2021/09/20/business/jeff-bezos-earth-fund.amp.html">Simple link #2</a></p>
+        <p class="expected">Expected: https://www.nytimes.com/2021/09/20/business/jeff-bezos-earth-fund.html</p>
+      </li>
+      <li>
+        <p><a href="https://www.google.fr/amp/s/www.brookings.edu/research/reforming-global-fossil-fuel-subsidies-how-the-united-states-can-restart-international-cooperation/%3famp">Non Standard TLD (Google Domain)</a></p>
+        <p class="expected">Expected: https://www.brookings.edu/research/reforming-global-fossil-fuel-subsidies-how-the-united-states-can-restart-international-cooperation</p>
+      </li>
+      <li>
+        <p><a href="https://www-wpxi-com.cdn.ampproject.org/v/s/www.wpxi.com/news/top-stories/pihl-bans-armstrong-student-section-hockey-games-after-vulgar-chants-directed-female-goalie/G2RP5FZA3ZDYRLFSWGDGQH2MY4/?amp_js_v=a6&amp_gsa=1&outputType=amp&usqp=mq331AQKKAFQArABIIACAw%3D%3D&fbclid=IwAR1TYTsjFEPy42VACpBR0n2jP-6whynOxwVqxpDMTWIRgW2vFlj573xVkEI#aoh=16358554203777&referrer=https%3A%2F%2Fwww.google.com&amp_tf=From%20%251%24s&ampshare=https%3A%2F%2Fwww.wpxi.com%2Fnews%2Ftop-stories%2Fpihl-bans-armstrong-student-section-hockey-games-after-vulgar-chants-directed-female-goalie%2FG2RP5FZA3ZDYRLFSWGDGQH2MY4%2F">ampproject.org with parameters</a></p>
+        <p class="expected">Expected: https://www.wpxi.com/news/top-stories/pihl-bans-armstrong-student-section-hockey-games-after-vulgar-chants-directed-female-goalie/G2RP5FZA3ZDYRLFSWGDGQH2MY4/?amp_js_v=a6&_gsa=1&outputType=amp&usqp=mq331AQKKAFQArABIIACAw%3D%3D&fbclid=IwAR1TYTsjFEPy42VACpBR0n2jP-6whynOxwVqxpDMTWIRgW2vFlj573xVkEI#aoh=16358554203777&referrer=https%3A%2F%2Fwww.google.com&_tf=From%20%251%24s&ampshare=https%3A%2F%2Fwww.wpxi.com%2Fnews%2Ftop-stories%2Fpihl-bans-armstrong-student-section-hockey-games-after-vulgar-chants-directed-female-goalie%2FG2RP5FZA3ZDYRLFSWGDGQH2MY4%2F</p>
```

Comment:
this is still an amp page (`<html amp=true`), although not served by google, is that expected?

Reply:
Kind of yes? This triggers the simple extraction. We're not recursively checking the links in case we get a double AMP link. I'll look in to the feasibility of this and update the page if needed.